app/controllers/pdl.py

```diff
@@ -267,21 +246,6 @@ def titles_title_i_state_distribution_search():
     return endpoint_response
 
 
-# /pdl/titles/title-i/subtitles/subtitle-a/map:
```

Comment:
I believe these eliminated endpoints are now handled by other database endpoints, right? Can you confirm @sandeep-ps ?